vitrina/settings.py

```diff
@@ -223,7 +223,8 @@
 SITE_ID = 1
 X_FRAME_OPTIONS = 'SAMEORIGIN'
 CMS_TEMPLATES = [
-    ('page.html', _("Home page")),
+    ('pages/page.html', _("Puslapis be šoninio meniu")),
+    ('pages/page_with_side_menu.html', _("Puslapis su šoniniu meniu"))
```

Comment:
This does not work, when selected.

![image](https://user-images.githubusercontent.com/297583/205019907-b8bc7467-20ab-4d43-a02a-a5118d03f060.png)

Reply:
Same as mentioned above, you need to add side menu plugin.